tests/controllers/test_crop_controller.py

```diff
@@ -50,45 +49,8 @@ def setUp(self):
         self.expected_weed_list = []
         self.expected_material_name = ["stage1", "stage2", "stage3", "stage4", "stage5",
                                        "stage6", "stage7", "stage8", "stage9", "stage10"]
-        self.num_crops_per_stage = {
-            "stage10": 0,
-            "stage8": 0,
-        }
 
 
-    def test_stage_crop_num_correct(self):
```

Comment:
is this test not required anymore?

Reply:
It's not required anymore as each stage is calculated by growth stage instead. The number won't be fixed.